composable_kernel/include/tensor_operation/gridwise_batched_gemm_xdlops_v2r3.hpp

```diff
@@ -0,0 +1,725 @@
+#ifndef CK_GRIDWISE_GEMM_XDLOPS_V2R3_SPLITM_HPP
+#define CK_GRIDWISE_GEMM_XDLOPS_V2R3_SPLITM_HPP
+
+#include "common_header.hpp"
+#include "multi_index_transform_helper.hpp"
+#include "tensor_descriptor.hpp"
+#include "tensor_descriptor_helper.hpp"
+#include "blockwise_gemm_xdlops.hpp"
+#include "blockwise_tensor_slice_transfer_v4r1.hpp"
+#include "threadwise_tensor_slice_transfer.hpp"
+#include "threadwise_tensor_slice_set.hpp"
+
+namespace ck {
+
+#if CK_EXPERIMENTAL_PASS_TENSOR_DESCRIPTOR_BY_VALUE
+template <typename GridwiseGemm,
+          typename FloatAB,
+          typename FloatC,
+          typename AGridDesc_B_K0_M_K1,
+          typename BGridDesc_K0_N_K1,
```

Comment:
If this is a "batched GEMM" then A, B, C matrices all need to have dimension "B"

Reply:
Don't have a better name for it. Suggestions?

Reply:
added dimension B for B matrix.